### DISCUSSION TITLE
2 issues when i start comfyui  with a very long time for creating images

### DISCUSSION THREAD

**Discussion by hemazone** [Comfy-Org/ComfyUI#4670] [Q&A]
hello , 

today i tried to open comfyui but got this 
`E:\ComfyU\python_embeded\Lib\site-packages\kornia\feature\lightglue.py:44: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.
  @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)`

and also this 

`E:\ComfyU\python_embeded\Lib\site-packages\transformers\tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884
  warnings.warn(
clip missing: ['text_projection.weight']
Requested to load FluxClipModel_`

and it was take  up to 3 min's to create image  according to my pc capabilities  now it take more than 50 min 

P.S 
i made # update the ComfyUI code
# update ComfyUI with the python dependencies

but still same  :( 
![Untitleءءءءءd](https://github.com/user-attachments/assets/70823329-5367-4d70-85ae-4af8d969a949)

**Comment by ltdrdata** [comment]
Reduandant issue:
https://github.com/comfyanonymous/ComfyUI/issues/4663#issuecomment-2315921154

**Reply by hemazone** [reply]
thank you bro. for your replay , but i did'nt get it , im still new here ,  do you mean its a bug and will be fixed soon or what ?

**Reply by ltdrdata** [reply]
You need to downgrade torch to 2.3.1.
torch 2.4.0 has that issue on windows.

**Reply by hemazone** [reply]
how to downgrade it ?

**Reply by ltdrdata** [reply]
```
E:\ComfyU\python_embeded\python.exe -m pip uninstall torch torchvision torchaudio
E:\ComfyU\python_embeded\python.exe -m pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cu121
```

**Reply by hemazone** [reply]
i'm sorry , i didn't read that  until now
i made mistake , didn't uninatall 2.4.0   and   installed  2.3.1     ( 2.7 GB  )
after it done , i found that it still   2.4.0  exist ,   , is there anyway to uninstall 2.4.0  without  reinstall 2.3.1 again ,  ??

**Reply by GTP98** [reply]
Hi I have the same problem installed the 2.3.1 but issue is still there.. Can you kindly share if you were you able to find a solution for this

**Comment by linkally** [comment]
I have the same issue, so i downgrade and insyall the 2.31,but the problem still there.
![Snipaste_2024-08-30_14-23-59](https://github.com/user-attachments/assets/7c01c025-59e0-4392-84a3-a67d30efd963)

**Reply by ltdrdata** [reply]
That's just the library developer changing specific behaviors through upgrades, causing a flood of reports about lazy responses to developers using that library, annoying them into making fixes.

The problem is that while such issues can sometimes be fixed by the tool creators, in cases where the tool's dependencies need to address it, there's no solution until that library provides an update.

TL;DR: Users can just ignore it.

**Reply by GTP98** [reply]
For me it takes around 2100s to generate a image. Can this be due to this error?
